tekton/pipelines/install.yaml

```diff
@@ -857,8 +857,11 @@ spec:
         - name: ibm_entitlement_key
           value: $(params.ibm_entitlement_key)
 
+        - name: cpd_product_version
```

Comment:
Doesn't there need to be some related changes in mascli/functions/xxx, or were those already done and this was just forgotten about?

Reply:
Let me move this to draft again... I will double check. I did that PR while working in other changes in devops/ansible-fvt, but did not check the existing flow in CLI... I will, before submitting to review again :+1: